src/__tests__/index.test.ts

```diff
@@ -440,6 +440,13 @@ const vectors: Array<TestVector> = [
       { text: '6bNNutYQz11WrkVCrj1nUS1dBGyoVZjdEg', hex: 'a914e613c7be9b53e1a47fd4edb3ea9777cf29dce30f87' },
     ],
   },
+  {
+    name: 'TFUEL',
+    coinType: 570,
```

Comment:
Is this correct coin type? 570 is allocated to BDX according to https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
I've opened a PR to add TFUEL to SLIP-0044: https://github.com/satoshilabs/slips/pull/1024
I will update this PR according to it when it finalizes.